src/client/wire_protocol/operations.rs

```diff
@@ -149,13 +192,13 @@ impl Message {
         let header = Header::with_query(total_length, request_id);
 
         Ok(Message::OpQuery { header: header, flags: flags,
-                              full_collection_name: full_collection_name,
+                              namespace: namespace,
```

Comment:
Sorry, I'm  getting confused because I literally just got a comment on this pull request telling me to make these changes.